src/gtimelog/utils.py

```diff
@@ -1,22 +1,139 @@
 """
 Misc utils
 """
+import datetime
+import os
+import re
+from gettext import gettext as _
 
-import sys
 
-import gi
+def as_minutes(duration):
+    """Convert a datetime.timedelta to an integer number of minutes."""
+    return duration.days * 24 * 60 + duration.seconds // 60
 
 
-def require_version(namespace, version):
+def as_hours(duration):
+    """Convert a datetime.timedelta to a float number of hours."""
+    return duration.days * 24.0 + duration.seconds / (60.0 * 60.0)
+
+
+def format_duration(duration):
+    """Format a datetime.timedelta with minute precision."""
+    h, m = divmod(as_minutes(duration), 60)
+    return '%d h %d min' % (h, m)
```

Comment:
Actually I can't find any place where `format_duration()` is called (outside of tests).  I think it can just go away?

Reply:
You're right. The other almost equivalent method need to be re-renamed i presume ;)